exercises/sgf-parsing/description.md

```diff
@@ -64,6 +64,13 @@ newlines any whitespace character encountered is replaced with a single space
 character irrespective of if the original character is found in its escaped
 from or its unescaped from.
 
+Be careful not to get confused between:
+
+- The string as it is represented in a string literal in the tests
+- The string that is passed to the SGF parser
+
+Escape sequences in the string literals may have already been processed by the programming language's parser before they are passed to the SGF parser.
```

Comment:
I don't like that I have to be wishy-washy with "may" here, but I can't speak for every language simultaneously because different languages may be able to show the strings differently, such as with some string literal that doesn't have escape sequences (some languages may call this a *raw string*).

It would also be great if I could show an example, but I also doubt I can do one that makes sense universally for all languages.

Reply:
Agree with the sentiment. The nature or prob-specs means we sometimes have to be somewhat _hand-wavy_, but this is still an improvement!